distribution/openrct2.d.ts

```diff
@@ -2495,11 +2495,25 @@ declare global {
          * Highest drop height in height units. Use `context.formatString()` to convert into metres/feet. Ex: `formatString('{HEIGHT}', ride.highestDropHeight)`.
          */
         readonly highestDropHeight: number;
+		
+		/**
+         * Set a breakdown on a ride.
+         * @param breakdown The type of breakdown to set.
+         */
+        setBreakdown(BreakdownType: breakdown): void;
+		
+		/**
+         * Fix a ride / clear the breakdown.
+         */
+        setBreakdown(): void;
```

Comment:
In the code its registered as `fixBreakdown`

Reply:
Ah, I forgot to hit save. I just did save-all in VS.. but the ts file was open in notepad++, sorry